### PR TITLE
Updated deprecated link in common-sso-google.rst

### PR DESCRIPTION
**Pull request by Rutam21**
#### Summary

Updated deprecated link in common-sso-google.rst

#### Ticket Link

NA

### CODE REVIEW

File: source/onboard/common-sso-google.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Migrating from OAuth 2.0 to OpenID Connect
 ------------------------------------------
 
-OAuth 2.0 is being deprecated and replaced by OpenID Connect. Refer to the product documentation to `convert your existing OAuth service provider configuration <https://docs.mattermost.com/cloud/cloud-administration/converting-oauth-2.0-to-openid-connect.html>`__ for Google Apps to the OpenID Connect standard.
+OAuth 2.0 is being deprecated and replaced by OpenID Connect. Refer to the product documentation to `convert your existing OAuth service provider configuration <https://docs.mattermost.com/onboard/cloud-converting-oauth-2.0-to-openid-connect.html>`__ for Google Apps to the OpenID Connect standard.
```
Comment (cwarnermm) [inline source/onboard/common-sso-google.rst:4]:
```suggestion
OAuth 2.0 is being deprecated and replaced by OpenID Connect. Refer to the product documentation to `convert your existing OAuth service provider configuration <https://docs.mattermost.com/onboard/convert-oauth20-service-providers-to-openidconnect.html>`__ for Google Apps to the OpenID Connect standard.
```

Comment (cwarnermm) [reply source/onboard/common-sso-google.rst:4]: I've provided an updated URL.

**Review by cwarnermm — COMMENTED** [review]
Thank you for this docs contribution. I've provided one updated URL.

**Comment by Rutam21** [comment]
@cwarnermm The changes have been applied. Please review and merge.

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**